src/redux/books/api.js

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
+import { booksFetched, bookAdded, bookRemoved } from './books';
+
+export const getBooks = () => async (dispatch) => {
```

Comment:
Please make server requests in the componentDidMount lifecycle method or with react hooks.